### ISSUE TITLE
ploteig calls /home/np29/bin/fixgreen

### ISSUE THREAD

**Issue by ddrichel** [DReichLab/EIG#13]

**Comment by muhammadsohailraza** [comment]
Hi, 
I use ploteig.pl to draw the plots from EIGENSOFT's example data, but i got following error, can anyone please help me to fix this issue:  Thanks!
command-line:
./ploteig2.pl -i example.eval -c 1:2 -p Case:Control -x -y -o sohail.plot

##Error: 
number of fields: 1
Case:Control
         line 0: warning: Skipping data file with no valid points
         line 0: warning: Skipping data file with no valid points

gnuplot> plot  "/leofs/zengchq_group/sohail/softwares/Eigensoft/EIG-6.1.4/TEST/example.eval.1:2:Case" using ($2)*1:($3)*1 title "Case" ,  "/leofs/zengchq_group/sohail/softwares/Eigensoft/EIG-6.1.4/TEST/example.eval.1:2:Control" using ($2)*1:($3)*1 title "Control" 
                                                                                                                                                                                                                                                                        ^
         line 0: x range is invalid

Can't exec "/home/np29/bin/fixgreen": No such file or directory at ./ploteig1.pl line 138, <FF> line 5.

**Comment by bumblenick** [comment]
1) Please comment out or remove the call to fixgreen
(My bad)
2) Please check very carefully that the last column of the evec file has
entries Case or Control
3) If 2) is OK make a file casecontrol with 2 lines Case aand Control and
change the Case:Control to caseconttrol

Sorry to put you through this.

Nick

**Comment by muhammadsohailraza** [comment]
Thank you @Nick!

**Comment by farnushfarhadi** [comment]
Hi, 

I am running in to the same problem :

**The command is**
>> smartpca.perl -i tmp/GenomTranscriptomTwinsMD_c1_QC_pruned_IBD.ped -a tmp/GenomTranscriptomTwinsMD_c1_QC_pruned_IBD.map -b tmp/GenomTranscriptomTwinsMD_c1_QC_pruned_IBD.fam -s 6 -e out/eval -l out/elog -o out/pca -p out/plot

**The error is**
>> ploteig -i out/pca.evec -c 1:2  -p Control  -x  -y  -o out/plot.xtxt
>> ## number of fields: 12
>> Control
>> Can't exec "/home/np29/bin/fixgreen": No such file or directory at /usr/local/bin/ploteig line 138, <FF> line 1984.
>> evec2pca-ped.perl 10 out/pca.evec tmp/GenomTranscriptomTwinsMD_c1_QC_pruned_IBD.fam out/pca

Shall I do the same thing you suggested above? I.e., commenting "system "/home/np29/bin/fixgreen  $psfile" ;" in **ploteig** file?

**Comment by bumblenick** [comment]
Yes please.   I will fix this soon.   gnuplot produces (or used to)
green that is nearly invisible when printed...

N

**Comment by farnushfarhadi** [comment]
Thank you. 
I have commented the corresponding line but still I get the error. Am I missing anything?

**Comment by bumblenick** [comment]
Well I think you must be.  If you comment out the call to fixgreen
you can't be calling it...

Nick
